doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,12 +24,25 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
+* Architectures:
+
+  * Fixed parsing of Cortex-M MemManage Stacking Errors, to correctly report thread stack corruptions.
+
 * Kernel:
 
   * The ``CONFIG_KERNEL_DEBUG`` Kconfig option, which was used to enable ``printk()`` based debugging of the kernel internals, has been removed.
     The kernel now uses the standard Zephyr logging API at DBG log level for this purpose.
     The logging module used for the kernel is named ``os``.
 
+* Boards:
+
+  * Introduced board definition for nRF5340 DK.
+  * Added the 'netif' capability for nRF52840 DK.
+  * Added GPIO as a supported feature in nRF-based board definitions.
```

Comment:
This could maybe be reworded to make it clear that it was a documentation oversight and GPIO was always supported, no?

Reply:
Well, I thought it is clear with "in nRF-based board definitions". Btw, this is not just documentation, right? It has an effect on sanity check filtering, no? But sure, it is nothing about the _real_ GPIO feature support :) 

I'll change "added" to "documented" to make this clear.